falcon_kit/functional.py

```diff
@@ -80,12 +80,14 @@ def LAsort_pair(line):
         dali2pairs[dali].add(pair)
     result = {}
     for dali, pairs in dali2pairs.items():
-        sorts = [pair2sort[pair] for pair in sorted(pairs, key=lambda k: (
+        sorted_pairs = sorted( pairs,  key=lambda k: (
             (int(k[0][1:]) if k[0].startswith('.') else 0),
             (int(k[1][1:]) if k[1].startswith('.') else 0)
-        ))]
+        ))
+        sorts = [pair2sort[pair] for pair in sorted_pairs]
+        checks = ["LAcheck {0}.db L1{1}{2}.las || exit 1".format(db_prefix, p1, p2)  for p1, p2 in sorted_pairs] 
```

Comment:
one more note, `LAcheck` does not find all errors, but that is a different issue.  (I manually permute and delete the bytes in a `las` file, but LAcheck fails to find the corruption.  I have not checked what exactly `LAcheck` does.)